content/terraform-docs-common/redirects.jsonc

```diff
@@ -164,15 +174,15 @@
     "destination": "/terraform/language/style",
     "permanent": true
   },
-  // HCP Terraform rebranding
   {
-    "source": "/terraform/cloud-docs/overview/estimate-terraform-cloud-cost",
-    "destination": "/terraform/cloud-docs/overview/estimate-hcp-terraform-cost",
+    "source": "/terraform/language/:version/syntax/style",
+    "destination": "/terraform/language/style",
     "permanent": true
   },
+  // HCP Terraform rebranding
   {
-    "source": "/terraform/language/settings/terraform-cloud",
```

Comment:
I moved all the `terraform/language/settings/terraform-cloud` redirects down together below!